Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -679,8 +679,8 @@
 "wikitext-upload-captcha-needed" = "Need CAPTCHA verification.";
 "wikitext-upload-result-unknown" = "Unable to determine wikitext upload result.";
 "wikitext-upload-save" = "Publishing...";
-"wikitext-upload-save-anonymously-or-login" = "Edits will be attributed to the IP address of your device. If you $1Log in$2 you will have more privacy.";
+"wikitext-upload-save-anonymously-or-login" = "Edits will be attributed to the IP address of your device. If you $1 Log in $2 you will have more privacy.";
 "wikitext-upload-save-anonymously-warning" = "Edits will be attributed to the IP address of your device. If you $1 you will have more privacy.";
 "wikitext-upload-save-sign-in" = "Log in";
-"wikitext-upload-save-terms-and-licenses" = "By publishing changes, you agree to the $1Terms of Use$2, and you irrevocably agree to release your contribution under the $3CC BY-SA 3.0$4 License and the $5GFDL$6. You agree that a hyperlink or URL is sufficient attribution under the Creative Commons license.";
```

Comment:
I was a little worried about this `$3CC BY-SA 3.0$4` string potentially causing problems for translators - hence this commit adding space (and the next one trimming it back out). I don't feel strongly about the issue given that our test should fail if the wrong number of substitution strings come back for a translation... happy to revert if you think this is overkill.

Reply:
Added “question” tag for the comment above...

Reply:
I think it's good to have available as an option if there is confusion from translators but to start let's keep it without spaces. I rolled back those commits for now.